invoices/invoices.go

```diff
@@ -656,6 +656,50 @@ type HtlcAcceptDesc struct {
 	AMP *InvoiceHtlcAMPData
 }
 
+// UpdateType is an enum that describes the type of update that was applied to
+// an invoice.
+type UpdateType uint8
```

Comment:
Q: Why wouldn't we want to add explicit structs for the corresponding updates? E.g. `CancelHtlcsUpdate` `SettleHodlInvoiceUpdate`